addons/website/models/res_config_settings.py

```diff
@@ -175,6 +175,7 @@ def action_open_robots(self):
             'type': 'ir.actions.act_window',
             "views": [[False, "form"]],
             'target': 'new',
+            'context': {'default_website_id': self.website_id.id}
```

Comment:
```suggestion
            'context': {'website_id': self.website_id.id}
```
remove everything else from the PR (except the js part)